pkg/console/tlsconfig/tlsconfig_test.go

```diff
@@ -266,3 +308,25 @@ func TestTlsConfig(t *testing.T) {
 	RegisterFailHandler(Fail)
 	RunSpecs(t, "TLS Config Suite")
 }
+
+// testCa is s self-signed certificate with that expires at 2033-07-23
+const testCa = `-----BEGIN CERTIFICATE-----
```

Comment:
Put this on top of the file?

Reply:
No, the same reason as in the other file.